proto/cheqd/v1/did.proto

```diff
@@ -7,7 +7,7 @@ import "google/protobuf/any.proto";
 import "cheqd/v1/common.proto";
 
 message Did {
-  repeated string context = 1; // optional
+  repeated string context = 1 [json_name="@context"]; // optional
   string id = 2;
   repeated string controller = 3; //optional
   repeated VerificationMethod verification_method = 4; // optional
```

Comment:
Just like the verification method, does it not make sense to have the following ones (L14-18) also as an object? These are all a set of Verification Method maps: https://w3c.github.io/did-core/#did-document-properties

Reply:
Also, this is likely going to tougher than what _this_ PR can achieve, so one to consider for the next sprint.

Reply:
I'm not sure that I correctly understood this idea. Let's discuss it on a call